### PR TITLE
[WIP] Add another Radarr Instance as a List (Integrates RadarrSync into Radarr directly)

### PR DESCRIPTION
**Pull request by galli-leo**
#### Database Migration
NO

#### Description

Threw this together rather quickly, but seems to work fine. Do we want to have an option to filter based on quality profiles of the other Radarr installation? Then we would have to load them in the UI though.

#### Todos
- [ ] Tests

#### Issues Fixed or Closed by this PR

* #3180

### CODE REVIEW

**Comment by Qstick** [comment]
Probably a good idea to add quality filtering. We should be able to grab as a list of profiles and pass to a select box in UI similar to hw v3 is grabbing pushbullet devices. Is there anything else we would want to do with filtering? (Only Import movies with xxx tag, or movies without xxx tag)

**Comment by Qstick** [comment]
Also where do you set the quality and rootfolder before sending list movies to movie service to be added? I didn’t see it in the sync handler in netimport search service. We don’t want to add movies with the quality and path they come from the other Radarr instance with. Maybe I’m missing it somewhere?

**Comment by galli-leo** [comment]
@Qstick Hmm I don‘t remember off the top of my head, but path and profile should be set somewhere :P

**Comment by edsai** [comment]
Anything new on this?

**Comment by simonk83** [comment]
Yep, keen for this too if possible?

**Comment by galli-leo** [comment]
@Qstick Do you think I should add some tests somewhere?

**Comment by galli-leo** [comment]
Also we should move the trakt authentication over to the system via RequestAction. Makes much more sense.

File: src/NzbDrone.Core/NetImport/RadarrInstance/RadarrInstanceSettings.cs [hunk 1]
```diff
@@ -0,0 +1,30 @@
+using NzbDrone.Core.Annotations;
+using NzbDrone.Core.ThingiProvider;
+using NzbDrone.Core.Validation;
+
+namespace NzbDrone.Core.NetImport.RadarrInstance
+{
+    public class RadarrInstanceSettings : IProviderConfig
+    {
+
```
Comment (Qstick) [inline src/NzbDrone.Core/NetImport/RadarrInstance/RadarrInstanceSettings.cs]: Should we add validation that URL, APIKey, Profile are not empty here

**Comment by Qstick** [comment]
You could add a test to ensure its parsed correctly. Add a json file with a sample response like we do with indexers and test that its parsed and de-serialized ok and then test that the profile >= 0 bit works in the parser file. That's about the only thing I can think of, short of the above validation comment

**Comment by millercentral** [comment]
Checking in on this one. Would love to use this approach instead of radarrsync.

**Comment by n1nj4888** [comment]
Any update on when this feature is likely to be incorporated into the nightly / develop branches?

**Comment by BEisem** [comment]
Would love to see this implemented as well. Any traction?

**Comment by beyondmeat** [comment]
Does #4062 supersed this?

**Comment by Qstick** [comment]
Yea for the most part, this is against develop, that is for Aphrodite. Don't know that we will be adding this to develop given its mostly just fixes now

**Pull request closed without merging**